HopsanCLI/main.cpp

```diff
@@ -895,20 +890,20 @@ int main(int argc, char *argv[])
                             {
                                 if (!line.empty())
                                 {
-                                    enabled_ports.push_back(line);
+                                    logOnlyPortsOrVariables.push_back(line);
```

Comment:
This variable name is really confusing, but I can't think of something better now.